src/main/java/com/mcmoddev/lib/exceptions/MaterialNotFoundException.java

```diff
@@ -0,0 +1,14 @@
+package com.mcmoddev.lib.exceptions;
+
+public class MaterialNotFoundException extends Exception {
+	
+	/**
+	 * 
+	 */
+	private static final long serialVersionUID = 1L;
+	
+	public MaterialNotFoundException(String materialName) {
+		super(materialName + " was not found in the tab array");

```

Comment:
The message appended could be more appropriate to reflect being a material instead of a tab.